gordo/server/views/base.py

```diff
@@ -240,13 +240,13 @@ def get(self):
 
 class ModelListView(Resource):
     """
-    List the current models capable of being served by the model
+    List the current models capable of being served by the server
     """
 
-    @api.doc(description="List the metadata for all models capable of being served.")
+    @api.doc(description="List the name of the models capable of being served.")
     def get(self, gordo_project: str):
         available_models = os.listdir(g.collection_dir)
-        return jsonify({"models": available_models})
+        return jsonify({"models": available_models, "revision": g.current_revision})
```

Comment:
May want to change the key to `current-revision` as there is a post request callback which appends `revision` to be the one that was asked for in the request.

Reply:
damndamndamn goodie point

Reply:
I just rely on that one instead of adding myself.